gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts

```diff
@@ -101,6 +107,8 @@ export class OrgSettingsUserDetailComponent implements OnInit, OnDestroy {
   applicationsTableDS: ApplicationDS[];
   applicationsTableDisplayedColumns = ['name'];
 
+  tokensDS: Token[];
```

Comment:
The `DS` suffix is for DataSource and is used to handled data inside a MatTable. Here I think we can get rid of 
```suggestion
tokens
```